codegen/configs/auth_employeeid.yaml

```diff
@@ -0,0 +1,5 @@
+package: web
```

Comment:
generate a strict server interface

Reply:
No, I did this first, but is pretty unusable for a http api.